.github/actions/testmo/testmo-init/action.yml

```diff
@@ -29,10 +33,12 @@ runs:
         ID=$(npx testmo automation:run:create \
           --instance "$TESTMO_URL" \
           --project-id 1 \
-          --name "Deployment tests" \
+          --name "$RUN_NAME" \
           --source frontend-e2e-tests)
         echo "TESTMO_RUN_ID=$ID" >> $GITHUB_OUTPUT
       env:
         TESTMO_URL: ${{ inputs.testmoUrl }}
         TESTMO_TOKEN: ${{ inputs.testmoToken }}
+        RUN_NAME: ${{inputs.testmoRunName}}
+        TAG: ${{inputs.tag}}
```

Comment:
`${{inputs.tag}}` isn't defined, no?

Reply:
Yes, I will remove TAG that's a leftovers